packages/infra/lib/api-stack/cw-enhanced-coverage-connector.ts

```diff
@@ -0,0 +1,239 @@
+import { Duration } from "aws-cdk-lib";
+import { SnsAction } from "aws-cdk-lib/aws-cloudwatch-actions";
+import { IVpc } from "aws-cdk-lib/aws-ec2";
+import { IFunction, ILayerVersion } from "aws-cdk-lib/aws-lambda";
+import * as secret from "aws-cdk-lib/aws-secretsmanager";
+import { Construct } from "constructs";
+import { getConfig } from "../shared/config";
+import { createScheduledLambda } from "../shared/lambda-scheduled";
+import { Secrets } from "../shared/secrets";
+
+export function settings(): {
+  connectorName: string;
+  scheduledLambda: {
+    scheduleExpression: string[];
+    memory: number;
+    timeout: Duration;
+  };
+  // chunkingLambda: {
+  //   memory: number;
+  //   batchSize: number;
+  //   // maxConcurrency: number;
+  //   timeout: Duration;
+  // };
+  // includeListLambda: {
+  //   // memory: number;
+  //   // batchSize: number;
+  //   // maxConcurrency: number;
+  //   // timeout: Duration;
+  // };
+  // sqsPatients: {
+  //   // HAS TO BE FIFO, cxId as the messageGroupId
+  //   // maxReceiveCount: number;
+  //   // visibilityTimeout: Duration;
+  //   // delayWhenRetrying: Duration;
+  // };
+  // sqsOrgChunks: {
+  //   // HAS TO BE FIFO, cxId as the messageGroupId
+  //   // maxReceiveCount: number;
+  //   // visibilityTimeout: Duration;
+  //   // delayWhenRetrying: Duration;
+  // };
+} {
+  // How long can the lambda run for, max is 900 seconds (15 minutes)
+  // const timeoutChunkingLambda = Duration.minutes(15).minus(Duration.seconds(5));
+  // const timeoutIncludeListLambda = Duration.minutes(15).minus(Duration.seconds(5));
+  return {
+    connectorName: "CWEnhancedCoverage",
+    scheduledLambda: {
+      scheduleExpression: ["10 * * * ? *"],
+      memory: 512,
+      // How long can the lambda run for, max is 900 seconds (15 minutes)
+      timeout: Duration.minutes(15).minus(Duration.seconds(5)),
+    },
+    // chunkingLambda: {
```

Comment:
// TODO 1195

Reply:
Leaving as is as these got updated on the downstream dependency